### DISCUSSION TITLE
Syntax Tips

### DISCUSSION THREAD

**Discussion by frioux** [community/community#10067] [Other Feature Feedback, Questions, & Ideas] [answered]
Love the new Code Search!  It would be really helpful if the syntax were included in brief on the search page, *especially* when there are no results.

**Comment by colinwm** [comment, marked as answer]
@frioux Great suggestion! I've added a short help guide when there are no results!

![image](https://user-images.githubusercontent.com/78169714/152552917-8bd4dc2a-3ca6-4b09-9cbf-cf85a9431481.png)
